### PR TITLE
Azure Blob Storage Collection Update release note

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create release note for Azure Blob Storage Collection Update.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Release notes for Azure Blob storage](https://sumologic.atlassian.net/browse/SUMO-240929)

### CODE REVIEW

File: blog-service/2024-02-09-apps.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+We're happy to announce an update of the Azure Blob Storage collection framework. Here are some of the key features that this update offers:
+
+- Aligns all the components to the latest version of Azure Functions runtime and libraries.
+- Replaces the soon-to-be deprecated classic application insights resource with new workspace-based application insights.
+- Updated function application logging with log levels.
+- Provides necessary security updates.
+
+For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). Also, for more information on metrics collection from Azure Monitor, see our [documentation](https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage/).
```
Comment (jpipkin1) [inline blog-service/2024-02-09-apps.md:21]:
```suggestion
For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). Also, for more information on metrics collection from Azure Monitor, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage/).
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. You'll need to change the date on the release note.

File: blog-service/2024-02-09-apps.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+We're happy to announce an update of the Azure Blob Storage collection framework. Here are some of the key features that this update offers:
+
+- Aligns all the components to the latest version of Azure Functions runtime and libraries.
+- Replaces the soon-to-be deprecated classic application insights resource with new workspace-based application insights.
+- Updated function application logging with log levels.
+- Provides necessary security updates.
+
+For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). Also, for more information on metrics collection from Azure Monitor, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage/).
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply blog-service/2024-02-09-apps.md:21]: thanks for confirming!

Comment (kimsauce) [inline blog-service/2024-02-09-apps.md:21]:
```suggestion
For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). For more information on logs collection from Azure Blob Storage, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage).
```

File: blog-service/2024-02-09-apps.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+We're happy to announce an update of the Azure Blob Storage collection framework. Here are some of the key features that this update offers:
+
+- Aligns all the components to the latest version of Azure Functions runtime and libraries.
+- Replaces the soon-to-be deprecated classic application insights resource with new workspace-based application insights.
+- Updated function application logging with log levels.
+- Provides necessary security updates.
+
+For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). For more information on logs collection from Azure Blob Storage, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage).
```
Comment (kimsauce) [inline blog-service/2024-02-09-apps.md:21]:
```suggestion
For upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). For information on collecting logs from Azure Blob Storage, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage).
```

**Review by kimsauce — APPROVED** [review]

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
Looks good, @JV0812. You'll need to change the date on the release note.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**